src/plugins/data/common/search/aggs/buckets/_terms_other_bucket_helper.test.ts

```diff
@@ -325,6 +325,43 @@ describe('Terms Agg Other bucket helper', () => {
         expect(typeof agg).toBe('function');
       });
 
+      test('returns a function for undefined agg buckets', () => {
+        const response = {
+          took: 10,
+          timed_out: false,
+          _shards: {
+            total: 1,
+            successful: 1,
+            skipped: 0,
+            failed: 0,
+          },
+          hits: {
+            total: 14005,
+            max_score: 0,
+            hits: [],
+          },
+          aggregations: {
+            2: {
+              doc_count_error_upper_bound: 0,
+              sum_other_doc_count: 8325,
+              buckets: [
```

Comment:
Is it correct that `buckets` has data here for a test which should check on missing buckets?